### DISCUSSION TITLE
Standup (Thursday November 24th)

### DISCUSSION THREAD

**Discussion by BrandonMagana** [BBGTC/food-bank-app#70] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Implemented auth logic for client admin

**What are your priorities for the next 24 hours?**:
 - Send a PR (it's getting big)

**Blockers**:
 - I have to finish my elective course assignments
 - Might visit my gf this weekend

**Shoutouts**:
 - @CarlosCRG19 and @BrandonMagana for attending class today :+1:

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Almost done with the news Screen

**What are your priorities for the next 24 hours?**:
 -  Finish screen and submit PR

**Blockers**:
 - Elective course homework

**Shoutouts**:
 - @DavBot02  for caring about his classmates and staying home

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Not too much, had problems with storage of access. token

**What are your priorities for the next 24 hours?**:
 - Finish inventory list

**Blockers**:
 - Time

**Shoutouts**:
 - @BrandonMagana and @antonieto for finishing the network assignment
